scripts/go-mod-tidy-all.sh

```diff
@@ -5,5 +5,5 @@ set -euo pipefail
 for modfile in $(find . -name go.mod); do
  echo "Updating $modfile"
  DIR=$(dirname $modfile)
- (cd $DIR; go mod tidy)
+ (cd $DIR; go mod tidy -go=1.20)
```

Comment:
please revert this,

Reply:
probably best to close the PR in that case -- it's the only meaningful change

Reply:
the logic is here:

https://github.com/cosmos/cosmos-sdk/issues/13976

1 cosmos-sdk should use 1 version of golang

Reply:
this is not how we enforce it, also we are a library not a binary. The application must enforce the go version